content/support-matrix/delfin/dell-emc/_index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Why are we putting these template? Above file you have already provided the steps for creating matrix and given template. Let the driver writer/document owner create these files

Reply:
Later those will be filled. Issue has been raised for the same #144